extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -270,6 +271,13 @@ public void onConnectionLost(HueBridge bridge) {
     public void onConnectionResumed(HueBridge bridge) {
         logger.debug("Bridge connection resumed. Updating thing status to ONLINE.");
         updateStatus(ThingStatus.ONLINE);
+        // now also re-initialize all light handlers
+        for (Thing thing : getThing().getThings()) {
+            ThingHandler handler = thing.getHandler();
+            if (handler != null) {
+                handler.initialize();
```

Comment:
> it was there before :-)

Right, so it had been added by accident and not removed by accident.

> there is AFAIK no other solution to ensure that the lights are initialized

The framework should always initialize a handler for an existing Thing - this must not be done by the binding. Check with @sbussweiler.